src/models/clients.models.ts

```diff
@@ -44,13 +56,22 @@ export function addClient(client) {
   return db
     .query(
       `INSERT INTO clients 
-            (passcode,time_passcode,time_passcode_expiry,gov_id,condition,deleted,phone,email,name,gender) 
-            VALUES ($1,$2,$3,$4,$5,$6,$7,$8,$9,$10)
-            RETURNING id`,
+      (passcode,gov_id,condition,phone,email,name,gender) 
+      VALUES ($1,$2,$3,$4,$5,$6,$7)
+      RETURNING id`,
       user
     )
     .then((govId) => {
       return govId.rows[0].id;
```

Comment:
`govId` is a misleading name here, just do 
```suggestion
    .then(({ rows }) => {
      return rows[0].id;
```